astrology.lic

```diff
@@ -46,14 +46,21 @@ class Astrology
   def do_buffs(settings)
     return unless settings
     return unless settings.waggle_sets['astrology']
-
+    
     @equipment_manager.empty_hands
     # Pop out rtr data from buffs and save it for later
     buffs = settings.waggle_sets['astrology']
     @rtr_data = buffs.select { |spell| spell.eql?('Read the Ripples') }
                      .values
                      .first
     buffs.reject! { |spell| spell.eql?('Read the Ripples') }
+    
+    buffs.keys.each{ |key| 
```

Comment:
I think rubocop will change this to `each_key`. Would you mind running rubocop on this? if you do `bundle install` while in the repository root directory it will install rubocop. Then you can do `mv astrology.lic astrology.rb` run rubocop in autocorrect mode, `rubocop -a astrology.rb` then `mv astrology.rb astrology.lic`. Alternatively you can do `rubocop -a` which will autocorrect all scripts.